app/models/general_content.rb

```diff
@@ -3,7 +3,11 @@ class GeneralContent < Node
 
   validates_with SectionHeritageValidator
   validates_presence_of :section
-  validates :parent, ancestry_depth: { minimum: 2 }
+  validates :parent, ancestry_depth: { minimum: 2 }, if: :not_root_page?
+
+  def not_root_page?
+    not section.is_a? RootSection
```

Comment:
use `!` not `not`. Keyword versions of the ruby operators have different precedence rules and a generally frowned upon.